### PR TITLE
Implement `DockerImage` `Cfgu type` in @configu/ts

### PR DESCRIPTION
**Pull request by Sadaf-A**
This PR adds DockerImage type to configu. 
as requested in #161

### CODE REVIEW

File: ts/packages/ts/src/ConfigSchema.ts [hunk 1]
```diff
@@ -56,6 +56,11 @@ export abstract class ConfigSchema implements IConfigSchema {
           validator.isHash(value, 'sha512'),
         Country: ({ value }) => validator.isISO31661Alpha2(value) || validator.isISO31661Alpha3(value),
         Currency: ({ value }) => validator.isISO4217(value),
+        DockerImage: ({ value }) => 
+          // eslint-disable-next-line no-useless-escape
+          /^(?:(([a-z0-9]|[a-z0-9][a-z0-9\\-]*[a-z0-9])\.)+([a-z0-9]|[a-z0-9][a-z0-9\\-]*[a-z0-9])(:[0-9]+\/)?(?:[0-9a-z-]+[/@])(?:([0-9a-z-]+))[/@]?(?:([0-9a-z-]+))?(?::[a-z0-9\\.-]+)?|([^:\/?#\s]+):\/\/(?:([^@\/?#\s]+)@)?([^\/?#\s]+)?(?:\/([^?#\s]*))?(?:[?]([^#\s]+))?\S*)$/gm.test(
```
Comment (rannn505) [inline ts/packages/ts/src/ConfigSchema.ts:61]: @Sadaf-A, did you craft this regex pattern yourself or draw inspiration from a source? Could you please provide a comment with either a link to a tester like https://regex101.com/ or the direct source link?

File: types/Cfgu.ts [hunk 2]
```diff
@@ -20,7 +20,8 @@ export type CfguType =
   | "Locale"
   | "LatLong"
   | "Country"
-  | "Currency";
+  | "Currency"
+  | "DockerImage";
```
Comment (rannn505) [inline types/Cfgu.ts:24]: Great work!

**Review by rannn505 — APPROVED** [review]
@Sadaf-A, I'm genuinely impressed with the implementation! I'm eager to continue our collaborative efforts on Configu. Please take a look at the minor comment I left. I've approved it for now. Great job! 🚀

File: types/Cfgu.ts (hunk 2 above)
Comment (Sadaf-A) [reply types/Cfgu.ts:24]: Thanks!

File: ts/packages/ts/src/ConfigSchema.ts (hunk 1 above)
Comment (Sadaf-A) [reply ts/packages/ts/src/ConfigSchema.ts]: @rannn505 I took reference from the `StackOverflow` link that you had provided in the issue itself.
I've updated the `RegExp` and below is the link for testing.
https://regex101.com/r/egVJW9/1 
Thank you,
Sadaf

**Pull request merged**